nncf/tensorflow/pruning/base_algorithm.py

```diff
@@ -173,6 +174,11 @@ def get_transformation_layout(self, model: tf.keras.Model) -> TFTransformationLa
 
             _, layer_info = converter.get_layer_info_for_node(spec_node.node_name)
             for weight_def in spec_node.metatype.weight_definitions:
+                if not layer.scale and weight_def.weight_attr_name == 'gamma':
```

Comment:
add also `if isinstance(spec_node.metatype, TFBatchNormalizationLayerMetatype) and ...` to avoid possible problems with other layers

Reply:
Thanks, fixed